.isort.cfg

```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=dateutil,openupgradelib,setuptools
```

Comment:
This change I guess is not needed?

Reply:
@alexey-pelykh Fixed, this is auto generate from pre-commit